helpers/queries/virtual-protocol-volume.sql

```
@@ -73,9 +82,41 @@ WITH
         AND l.block_time >= from_unixtime({{startTimestamp}})
-        AND l.block_time <= from_unixtime({{endTimestamp}})
+        AND l.block_time < from_unixtime({{endTimestamp}})
+    ),
+
+    sol_configs AS (
+        SELECT config
+        FROM meteora_solana.dynamic_bonding_curve_evt_evtcreateconfig
+        WHERE quote_mint = '3iQL8BFS2vE7mww4ehAqQHAsbmRNCrPxizWAT2Zfyr9y'
+
+        -- UNION, not UNION ALL: every config is emitted into both tables, so ALL doubles it
+        UNION
```

SQLFluff (4.3.0)

[error] 'UNION [DISTINCT|ALL]' is preferred over just 'UNION'.

(AM02)